src/test/java/org/apache/commons/validator/routines/IBANValidatorTest.java

```diff
@@ -309,95 +248,198 @@ private static String formatToRE(final String type, final int len) {
+            for (final CSVRecord o : p) {
+                final String item = o.get(0);
+                switch (item) {
+                    case "Name of country":
+                        country = o;
+                        break;
+                    case "IBAN electronic format example":
+                        electronicExample = o;
+                        break;
+                    default:
+                        break;
+                }
+            }
         }
-        System.out.println("Processed " + countries + " countries.");
+
+        assertNotNull(country);
+        assertNotNull(electronicExample);
+
+        Collection<Arguments> result = new ArrayList<>();
+        for (int i = 1; i < country.size(); i++) {
+            result.add(Arguments.of(country.get(i), electronicExample.get(i)));
+        }
+
+        return result;
     }
 
-    private static void printEntry(final String ccode, final String length, final String ib, final String country) {
-        final String fmt = String.format("\"%s\"", ib);
-        System.out.printf("            new Validator(\"%s\", %s, %-40s), // %s\n", ccode, length, fmt, country);
+    @ParameterizedTest
+    @MethodSource("ibanRegistryV98Source")
+    void validatorShouldExistWithProperConfiguration(String countryName, String countryCode, List<String> acountyCode, int ibanLength, String structure) throws Exception {
+        String countryInfo = " countryCode: " + countryCode + ", countryName: " + countryName;
+        Validator validator = IBANValidator.getInstance().getValidator(countryCode);
+
+        assertNotNull(validator, "IBAN validator returned null for" + countryInfo);
+        assertEquals(ibanLength, validator.getIbanLength(), "IBAN length should be " + ibanLength + " for" + countryInfo);
+
+        List<String> allPatterns = Arrays.stream(validator.getRegexValidator().getPatterns()).map(Pattern::pattern).collect(
+                Collectors.toList());
+
+        String re = fmtRE(structure.substring(2));
+        assertTrue(allPatterns.remove(countryCode + re), "No pattern " + countryCode + re + " found for " + countryInfo);
+        for (String ac : acountyCode) {
+            assertTrue(allPatterns.remove(ac + re), "No additional country code " + ac + " found for " + countryInfo);
+        }
+        assertTrue(allPatterns.isEmpty(), "Unrecognized patterns: " + allPatterns + " for" + countryInfo);
+    }
+
+    @ParameterizedTest
+    @MethodSource("ibanRegistryV98SourceExamples")
+    void exampleAccountsShouldBeValid(String countryName, String example) {
+        Assumptions.assumeFalse(INVALID_IBAN_FIXTURES.contains(example), "Skip invalid example: " + example + " for " + countryName);
+        assertTrue(IBANValidator.getInstance().isValid(example), "IBAN validator returned false for " + example + " for " + countryName);
     }
 
     @Test
-    public void testGetRegexValidatorPatterns() {
+    void testGetRegexValidatorPatterns() {
```

Comment:
Hello @slawekjaranowski 
Please don't make cosmetic changes, it makes it harder and takes longer to review a PR (FWIW, I don't agree with the JUnit style guideline on this kind of change). Ideally, a PR should only contain required changes. TY!

Reply:
ok, I will revert such

Reply:
@garydgregory  - done